routes.py

```diff
@@ -1,11 +1,14 @@
-from flask import Flask
+from flask import (
+    Flask,
+    render_template,
+)
 
 app = Flask(__name__)
 
 
 @app.route("/")
```

Comment:
Why the `@` symbol? Are we telling computer how to delineate folders in the path?

Reply:
It's a bit advanced, but they are called "Decorators" and they modify the behavior of a method or function. In this case they are letting flask know that this is the method to call when someone navigates to whatever path is in that string.